### PR TITLE
Fix - Organization checks for members

### PR DESCRIPTION
**Pull request by naveensrinivasan**
* **Please check if the PR fulfills these requirements**
- [x] Tests for the changes have been added (for bug fixes/features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bugfix for Org checks for members

* **What is the current behavior?** (You can also link to an open issue here)
The current check ignores the org the members are in

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
None

* **Other information**:

### CODE REVIEW

File: checks/contributors.go [hunk 1]
```diff
@@ -38,6 +43,28 @@ func Contributors(c checker.Checker) checker.CheckResult {
...
+			} else {
+				var orgs orgnaizations
+
+				err = json.NewDecoder(resp.Body).Decode(&orgs)
+
+				if err != nil {
+					c.Logf("unable to get  decode json for for %s org", contrib.GetLogin())
+				} else {
+					if len(orgs) > 0 {
+						companies[orgs[0].Login] = struct{}{}
```
Comment (inferno-chromium) [inline checks/contributors.go:61]: So, this does not do the filtering we do in the lines below [e.g. strip inc., llc, etc]. Probably need to move filtering to a helper function and then call in these 2 places.

**Review by inferno-chromium — APPROVED** [review]
LGTm with one small change needed.

File: checks/contributors.go [hunk 2]
```diff
@@ -38,6 +43,28 @@ func Contributors(c checker.Checker) checker.CheckResult {
...
+
+			if e != nil {
+				c.Logf("unable to get org members for %s", contrib.GetLogin())
+			} else {
+				var orgs orgnaizations
+
+				err = json.NewDecoder(resp.Body).Decode(&orgs)
+
+				if err != nil {
+					c.Logf("unable to get  decode json for for %s org", contrib.GetLogin())
```
Comment (inferno-chromium) [inline checks/contributors.go:58]: nit: 2 spaces after get. two repeated fors, also it is user for %s user or just %s.

**Review by inferno-chromium — APPROVED** [review]
LGTm with one small change needed.

File: checks/contributors.go [hunk 3]
```diff
@@ -38,6 +43,28 @@ func Contributors(c checker.Checker) checker.CheckResult {
...
+				} else {
+					if len(orgs) > 0 {
+						companies[orgs[0].Login] = struct{}{}
+						continue
```
Comment (inferno-chromium) [inline checks/contributors.go:62]: Also, why continue, i think we need to check company as well.

**Review by inferno-chromium — APPROVED** [review]
One more change needed, see comment.

File: checks/contributors.go (hunk 3 above)
Comment (naveensrinivasan) [reply checks/contributors.go:62]: Your org's is not shown, because you don't have any orgs that are public.

Comment (naveensrinivasan) [reply checks/contributors.go:62]: We shouldn't count for the same contributor in `org` and `company` and that is the reason for continue. Here is my id https://api.github.com/users/naveensrinivasan/orgs , I had to enable the public settings in every org I am in.

File: checks/contributors.go (hunk 1 above)
Comment (naveensrinivasan) [reply checks/contributors.go:61]: Took care of it. Thanks

File: checks/contributors.go (hunk 2 above)
Comment (naveensrinivasan) [reply checks/contributors.go:58]: Took care of it.

**Comment by inferno-chromium** [comment]
E.g. check https://api.github.com/users/dlorenc/orgs vs https://github.com/dlorenc 
https://github.com/dlorenc shows openssf org, whereas https://api.github.com/users/dlorenc/orgs does not. So, might need to consider both??

Also, i dont see filtering logic moved to helper function. maybe github org name does not need filtering (now that i think about it, it can't have llc, inc, etc).

**Comment by naveensrinivasan** [comment]
🤦 My commit got messed up while rebasing on the refactoring.

**Comment by naveensrinivasan** [comment]
> E.g. check https://api.github.com/users/dlorenc/orgs vs https://github.com/dlorenc
> https://github.com/dlorenc shows openssf org, whereas https://api.github.com/users/dlorenc/orgs does not. So, might need to consider both??
> 
You are able to see `dlorenc` in `openssf` because you are also in that org. Or else you will not be able to see the org private status

 
image

**Comment by inferno-chromium** [comment]
> 🤦 My commit got messed up while rebasing on the refactoring.

No worries Sir!, can fix this sometime next week, enjoy long weekend.

> > E.g. check https://api.github.com/users/dlorenc/orgs vs https://github.com/dlorenc
> > https://github.com/dlorenc shows openssf org, whereas https://api.github.com/users/dlorenc/orgs does not. So, might need to consider both??
> 
> You are able to see `dlorenc` in `openssf` because you are also in that org. Or else you will not be able to see the org private status

You are so right! cant see this openssf from incognito, so no change needed.

**Pull request merged**